doc/rados/operations/health-checks.rst

```diff
@@ -1209,3 +1212,6 @@ This warning can silenced by setting the
 ``mon_warn_on_osd_down_out_interval_zero`` to false::
 
   ceph config global mon mon_warn_on_osd_down_out_interval_zero false
+
+.. _Storage Capacity: ../configuration/mon-config-ref/#storage-capacity
```

Comment:
these links are not reachable. see https://ceph--38145.org.readthedocs.build/en/38145/rados/operations/health-checks/#pool-near-full

instead of using hyperlinks, i'd suggest adding labels using `:ref:` . see https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations

for two reasons:

- simpler this way. you can save the typing for adding the text for the link. as the section title is used for the label referencing it.
- it is future proof. even if the file is moved around. the label is not changed. but the relative path changes accordingly.

Reply:
I'd tried that first, I couldn't get it to pass the Jenkins tests.  I will revisit.